domain/schema/controller.go

```diff
@@ -279,3 +280,30 @@ CREATE TABLE model_list (
 );
 `[1:]
 }
+
+func controllerConfigSchema() string {
+	return `
+CREATE TABLE controller_config (
+    key     TEXT PRIMARY KEY,
+    value   TEXT
+);
+CREATE TRIGGER trg_log_controller_config_insert
+AFTER INSERT ON controller_config FOR EACH ROW
+BEGIN
+    INSERT INTO change_log (edit_type_id, namespace_id, changed_identifier, created_at) 
```

Comment:
That's what we'll call it, we've not moved to it yet. That'll come. For now stick to changed_uuid for all triggers.

```suggestion
    INSERT INTO change_log (edit_type_id, namespace_id, changed_uuid, created_at) 
```